06_gpu_and_ml/comfyui/comfyapp.py

```diff
@@ -235,10 +235,10 @@ def poll_server_health(self) -> Dict:
 # ![comfyui menu](./comfyui_menu.jpeg)
 
 # ## More resources
+# - Use memory snapshotting to [speed up ComfyUI cold starts](/blog/comfyui-mem-snapshots)
+# - Run a ComfyUI workflow as a [Python script](/blog/comfyui-prototype-to-production)
 
-# - Run a ComfyUI workflow as a [Python script](https://modal.com/blog/comfyui-prototype-to-production)
-
-# - When to use [A1111 vs ComfyUI](https://modal.com/blog/a1111-vs-comfyui)
+# - When to use [A1111 vs ComfyUI](/blog/a1111-vs-comfyui)
 
 # - Understand tradeoffs of parallel processing strategies when
-# [scaling ComfyUI](https://modal.com/blog/scaling-comfyui)
```

Comment:
We prefer making the links absolute in the examples repo, so that they work in an editor.